itest/integration_test.go

```diff
@@ -56,34 +56,31 @@ func TestTaprootAssetsDaemon(t *testing.T) {
 			testCase.name)
 
 		success := t.Run(testCase.name, func(t1 *testing.T) {
-			// Create a subtest harness for each test case.
-			subTestLnd := lndHarness.Subtest(t1)
-
```

Comment:
But ideally, don't we want a new LND node for each itest? This is encase LND might hold some state across tests and erroneously cause a test to pass (via false positive).

Reply:
At the moment the only thing we require of `lnd` is it having some funds and it tracking keys and addresses. So there is no state that can conflict between the tests. And as long as that is the case, I think we should take the quite significant speedup we get from not restarting the nodes.